lib/workload/components/icav2-copy-files/check_or_launch_job_lambda_py/check_or_launch_job_lambda.py

```diff
@@ -50,8 +50,12 @@
     from mypy_boto3_secretsmanager import SecretsManagerClient
 
 # Set logging
-logging.basicConfig(level=logging.INFO)
-logger = logging.getLogger(__name__)
+logging.basicConfig(
+    level=logging.INFO,
+    force=True,
```

Comment:
This will remove any preconfigured handlers... 
Is that what we want? 

The accepted answer claims that the default AWS handler "might also add some metadata to the record if available", which sounds like it could be useful? 

Happy to accept the change. Longer term we should probably switch to the lambda power tools...

Reply:
Oh those docs are actually very handy!

Hmm, yes you might be right, from this below, the logger may in fact set the requestId as well.  

```
Using structured JSON logs with Python
If you select JSON for your function’s log format, Lambda will send logs output by the Python standard logging library to CloudWatch as structured JSON. Each JSON log object contains at least four key value pairs with the following keys:

"timestamp" - the time the log message was generated

"level" - the log level assigned to the message

"message" - the contents of the log message

"requestId" - the unique request ID for the function invocation
```

We could rearrange from 

```
import logging
logging.basicConfig(
    level=logging.INFO,
    force=True,
    format='%(asctime)s %(message)s'
)
logger = logging.getLogger()
```

To just

```
import logging
logger = logging.getLogger()
logger.setLevel(logging.INFO)
```

And then we still get the same log levels, just lose the formatting a bit

Reply:
If that works locally... 
I think the issue was that then on the local env there may not be a log handler at all, hence the fallback to the `basisConfig` if no handler was found.

Reply:
> I think the issue was that then on the local env there may not be a log handler at all,

Yes, the answer that looks for handlers didn't seem very fun. 

Something worth considering is this too - https://docs.powertools.aws.dev/lambda/python/latest/core/logger/

Reply:
Yes, the power tools logging support. That's what I meant earlier.
(but there are also other reasons to use the power tools)